Tests/LibCrypto/CMakeLists.txt

```diff
@@ -18,3 +18,5 @@ set(TEST_SOURCES
 foreach(source IN LISTS TEST_SOURCES)
     serenity_test("${source}" LibCrypto LIBS LibCrypto)
 endforeach()
+
+target_link_libraries(TestRSA PRIVATE LibTLS)
```

Comment:
@alimpfard is there a LibTLS Test file that fits better than this one for this functionality? (decoding PublicKeyInfo + PKCS#8)

Reply:
Not really, all the tests used to be in test-crypto anyway
Maybe make a new test file?

Reply:
Moving to TestTLSCertificateParser.cpp, since the filename seems appropriate and it's also testing Certificate.h